src/main/resources/assets/mmorpg/lang/zh_cn.json

```diff
@@ -918,488 +862,544 @@
 	"mmorpg.advancement.desc.currencybag": "通货袋可以自动拾取挖矿与砍杀的矿石、通货等",
-	"mmorpg.advancement.desc.factorystation": "将不好的装备变成更稀有的物品.",
+	"mmorpg.advancement.desc.factorystation": "将垃圾装备转化为高稀有度物品。",
 	"mmorpg.advancement.desc.killspecificboss": "前往冒险地图并杀死这个特定的BOOS.",
-	"mmorpg.advancement.desc.levelpenalty": "当你杀死一个太高或太低的怪物时, 物品和经验将会减少. 尝试击杀和你等级相差5级以内的怪物.",
-	"mmorpg.advancement.desc.levelup": "每个等级的里程碑都允许使用新的以及更好的词缀、套装等物品的生成!获取新的战利品来提升自己",
-	"mmorpg.advancement.desc.levelup10": "从10级开始, 怪物将会开始掉落通货和冒险地图(默认配置).",
-	"mmorpg.advancement.desc.lootbag": "战利品袋可以自动拾取装备、技能和符文等",
+	"mmorpg.advancement.desc.levelpenalty": "当你杀死一个等级太高或太低的怪物时，你获得的物品和经验将会减少。尝试击杀和你等级相差5级以内的怪物。",
+	"mmorpg.advancement.desc.levelup": "每踏入一道新的等级里程碑，你获得的物品都会解锁出一些更强的的词缀、套装等。开始搜刮吧！",
+	"mmorpg.advancement.desc.levelup10": "从10级开始，怪物将会开始掉落通货和冒险地图（默认配置）。",
+	"mmorpg.advancement.desc.lootbag": "战利品袋可以自动拾取装备、法术和符文等",
 	"mmorpg.advancement.desc.mapbag": "地图袋可以自动拾取冒险地图",
-	"mmorpg.advancement.desc.mapdevice": "此设备允许你献祭冒险地图和种子来获取时间, 允许你放入冒险地图来获得特殊物品和通货的特殊奖励!",
-	"mmorpg.advancement.desc.masterbag": "大师袋可以自动拾取几乎所有挖矿与砍杀模组掉落的战利品. 此外还拥有物品分类功能.",
-	"mmorpg.advancement.desc.mobrarityspawnat": "这个品级怪物只有在一定等级后才会生成, 默认情况在以下等级之后: ",
-	"mmorpg.advancement.desc.modifystation": "装备强化太可用于使用通货修改装备. 通货有特殊效果, 例如添加前缀, 再次添加后缀等. 装备强化台还可以镶嵌符文和觉醒符文等.",
-	"mmorpg.advancement.desc.orboftransmutation": "完全重新创造一个物品, 让它有机会获得更高的品级. 最普遍的通货之一.",
-	"mmorpg.advancement.desc.repairstation": "装备修理台可用于修复你的装备. 将矿石(普通、下品等)放入燃料槽中, 然后放入物品进行修复. 如果放入能效珠将降低修复时燃料的消耗",
-	"mmorpg.advancement.desc.salvagestation": "将物品变成矿石, 有时也会变成通货.",
-	"mmorpg.advancement.desc.stoneofhope": "完全重新创造物品, 给与你一个更高的物品. 如果你有一个精品的装备拥有不好的属性这将非常有用.",
-	"mmorpg.advancement.desc.welcomemineandslash": "这个模组是关于击杀怪物来提升装备并杀死更高级的怪物!",
-
-"_comment": " [CATEGORY]: ADVANCEMENT DESCRIPTIONS - NAMES",
-
-
-"_comment": " [CATEGORY]: UNIQUE ITEMS - NAMES",
-
-	"item.mmorpg.uniques.axe.axefire0": "§e烈焰斧",
-	"item.mmorpg.uniques.axe.axethunder0": "§e雷电斧",
-	"item.mmorpg.uniques.axe.axewaterfire0": "§e冰霜斧",
-	"item.mmorpg.uniques.axe.axewind0": "§e飓风斧",
-	"item.mmorpg.uniques.boots.bootsfire0": "§e燃烧靴子",
+	"mmorpg.advancement.desc.mapdevice": "向此设备献祭冒险地图便可创建地牢，从地牢中你可以搜刮到一些特殊的战利品，例如珍品装备以及各种通货等。地图的等阶越高，所获得的奖励便越好！",
+	"mmorpg.advancement.desc.masterbag": "大师袋可以自动拾取几乎所有挖矿与砍杀模组掉落的战利品。此外它还拥有物品分类功能。",
+	"mmorpg.advancement.desc.mobrarityspawnat": "这个品级怪物只有在一定等级后才会生成，默认情况在以下等级之后: ",
+	"mmorpg.advancement.desc.modifystation": "在装备改造台中，你可以使用通货改造你的装备。通货有特殊效果，例如添加前缀，刷新后缀属性数值等。装备改造台还可以为你的装备镶嵌符文和觉醒符文之语等。",
+	"mmorpg.advancement.desc.orboftransmutation": "将凡品装备转化为下品装备。最常见的下品通货物品之一。",
+	"mmorpg.advancement.desc.repairstation": "装备修理台可用于修复你的装备。将矿物（下品、中品等）或是原版矿物放入燃料槽中，然后放入物品进行修复。如果放入能效珠，还可降低修复时燃料的消耗",
+	"mmorpg.advancement.desc.salvagestation": "将物品变成矿物以及通货。",
+	"mmorpg.advancement.desc.stoneofhope": "完全重新创造物品，且新物品的品级会提升。如果你有一件精品装备，但上面的属性不好或是你不需要，此通货便能帮助你重新刷新这一装备的属性。冒险地图中会掉落此物品，但需要地图的等阶至少为2。",
+	"mmorpg.advancement.desc.welcomemineandslash": "杀死怪物，获取强力装备，再用这些装备杀死更多的怪物。这就是本模组的玩法！",
```

Comment:
The old translations of advancement desc are so hard to read fluently. maybe it used machine translation?